src/app/components/controls-module/token-keyword-filter-control/token-keyword-filter-control.component.ts

```diff
@@ -26,6 +26,8 @@ export class TokenKeywordFilterControlComponent implements OnInit, OnDestroy {
   filter: TokensFilter;
   icons = Icons;
   IconSizes = IconSizes;
+  tokenOrderByTypes = TokenOrderByTypes;
+  tokenProvisionalTypes = TokenProvisionalTypes;
```

Comment:
If these aren't used in the template, can just set the type with `1tokenOrderByTypes: TokenOrderByTypes`

Reply:
Done